src/Bicep.LangServer.IntegrationTests/DefinitionTests.cs

```diff
@@ -60,7 +61,21 @@ public async Task GoToDefinitionRequestOnValidSymbolReferenceShouldReturnLocatio
                 link.TargetUri.Should().Be(uri);
 
                 // target range should be the whole span of the symbol
-                link.TargetRange.Should().Be(symbol.DeclaringSyntax.Span.ToRange(lineStarts));
+                try
+                {
+                    // For a nested resource the returned span info is *clamped* to the 0-th column for some reason I don't understand.
+                    // link.TargetRange == "[start: (270, 0), end: (286, 1)]"
+                    // symbol.DeclaringSyntax.Span.ToRange(lineStarts) == "[start: (273, 2), end: (285, 3)]"
+                    //
+                    // This is a nested resource and it's indented 2 columns in the source, so the language server is doing a wrong.
+                    // Need help with this :-/
```

Comment:
>this [](start = 38, length = 4)

Let's grab some time and dig into it.

Reply:
The root case was that `IntegrationTestHelper.GetPosition` wasn't generic enough. The test was doing things like a GTD on the LHS of `foo:bar` when it was supposed to be looking at `:bar` which obviously takes you to a different symbol 😆